docs-2.0/nebula-dashboard-en/3.create-import-dashboard/2.import-cluster.md

```diff
@@ -0,0 +1,32 @@
+# 导入集群
+
+本文介绍如何通过Dashboard导入集群，暂不支持导入使用Docker和Kubernetes方式部署的集群。
+
+## 操作步骤
+
+1. 在集群列表页面，点击 **导入集群** 标签。
+2. 在导入集群页面，输入连接Nebula Graph的信息：
+  - Graphd Host：<其中一个Graphd进程的虚机IP>:<端口号>。本示例设置为 `192.168.8.157:9669`。
+  - 用户名：连接Nebula Graph的账号。
+  - 密码：连接Nebula Graph的密码。
+
+  !!! note
+
+        因为 Nebula Graph 默认不启用身份验证，所以，一般情况下用户可以使用 `root` 账号和任意密码连接 Nebula。
```

Comment:
@kqzh If root is required or recommended, review it?

Reply:
root is required in this version, do we need enable_authorize=ture as a default? @NicolaCage